src/uwtools/rocoto.py

```diff
@@ -0,0 +1,44 @@
+"""
+Support for writing a Rocoto XML using J2Templating
+"""
+
+from uwtools.j2template import J2Template
+from uwtools.config import YAMLConfig
+
+# private functions
```

Comment:
`# Private functions`, and `# Main function` to `# Public functions` down below, for consistency with other modules, please.